content/pl/toc.md

```diff
@@ -0,0 +1,38 @@
+The Twelve Factors
+==================
+
+## [I. Codebase](./codebase)
+### Jedno źródło kodu śledzone systemem kontroli wersji, wiele wdrożeń
+
+## [II. Zależności](./dependencies)
+### Jawnie zadeklaruj i wydziel zależności
+
+## [III. Konfiguracja](./config)
+### Trzymaj konfigurację w środowisku
+
+## [IV. Usługi wspierające](./backing-services)
+### Traktuj usługi wspierające jako przydzielone zasoby
+
+## [V. Buduj, wypuszczaj, uruchamiaj](./build-release-run)
+### Precyzyjnie oddziel etap budowania od uruchamiania
+
+## [VI. Procesy](./processes)
+### Odpal aplikację jako jeden lub więcej bezstanowych procesów
+
+## [VII. Przydzielanie portów](./port-binding)
+### Eksportuj usługi przez przydzielenie portów
```

Comment:
To *eksportuj* tutaj średnio mi pasuje, bo tak naprawdę chodzi o *wystawianie na zewnątrz*, ale to brzmi jeszcze gorzej. Nie mam innego pomysłu, ale postaraj się znaleźć coś innego.